Back/Reciplastk.Gateway/Controllers/RolController.cs

```diff
@@ -0,0 +1,133 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Reciplastk.Gateway.DataAccess;
+using Reciplastk.Gateway.Models;
+using Reciplastk.Gateway.Services;
+
+namespace Reciplastk.Gateway.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class RolController : ControllerBase
```

Comment:
Controller name is incorrect. 
```suggestion
    public class RoleController : ControllerBase
```